src/main/java/edu/cornell/kfs/sys/service/impl/DocumentMaintenanceServiceImpl.java

```diff
@@ -46,8 +48,7 @@ public boolean requeueDocuments() {
     @Transactional
     private void requeueDocumentByDocumentId(List<ActionItemNoteDetailDto> noteDetails, String docId) {
         LOG.info("requeueDocumentByDocumentId: Requesting requeue for document: " + docId);
-        DocumentRefreshQueue documentRequeuer = (DocumentRefreshQueue) SpringContext.getService(CUKFSConstants.DOCUMENT_REFRESH_QUEUE_SERVICE_SPRING_CONTEXT_NAME);
-        documentRequeuer.refreshDocument(docId);
+        documentRefreshQueue.refreshDocument(docId);

```

Comment:
It looks like DOCUMENT_REFRESH_QUEUE_SERVICE_SPRING_CONTEXT_NAME is not longer used in the code base. Should this constant also be removed from CUKFSConstants?

Reply:
I'll remove the unused constant.

Reply:
Thank you.